detect_secrets/audit/common.py

```diff
@@ -91,6 +92,7 @@ def get_raw_secrets_from_file(
             line_numbers = list(range(len(lines_to_scan)))
 
         for line_number, line in zip(line_numbers, lines_to_scan):
+            context = get_code_snippet(lines=lines_to_scan, line_number=line_number)
```

Comment:
I'd use `line_getter.lines` for the context here because verify method on some plugins search around the secret for other needed variables (e.g. username), and lines_to_scan can potentially be the secret line alone (if secrets.line_number yields True). 

I think -I could be wrong- we should increment line_number by one similar to the plugin.analyze_line call below? I think the value of line_number here refers to the index in the list and no the actual line number in the file.

Reply:
Yes you are correct. Nice catch.